packages/launchpad/src/setup/ManualInstall.vue

```diff
@@ -22,12 +22,23 @@
           class="h-24px my-8px ml-24px w-24px float-right"
           :aria-label="t('setupPage.install.pendingInstall')"
         />
-        <ExternalLink
-          :href="`https://www.npmjs.com/package/${dep.package}`"
-          class="text-indigo-500 text-14px hocus-link-default"
-        >
-          {{ dep.package }}
-        </ExternalLink>
+        <span class="text-14px ">
+          <ExternalLink
+            :href="`https://www.npmjs.com/package/${dep.package}`"
+            class="text-indigo-500 hocus-link-default"
+          >
+            {{ dep.package }}
+          </ExternalLink>
+          <span class="rounded-md border border-gray-400 ml-6px">
+            <span class="p-4px">{{ dep.minVersion }}</span>
+            <span
+              v-if="!dep.satisfied && dep.detectedVersion"
+              class="border-l border-gray-400 bg-gray-100 text-gray-600"
+            >
+              Detected {{ dep.detectedVersion }}
+            </span>
+          </span>
```

Comment:
```suggestion
          <span class="border rounded-md font-semibold border-gray-100 text-xs ml-6px text-gray-700">
            <span class="p-4px">{{ dep.minVersion }}</span>
            <span
              v-if="!dep.satisfied && dep.detectedVersion"
              class="border-l font-medium bg-gray-50 border-gray-100 px-4px text-gray-600"
            >
              detected {{ dep.detectedVersion }}
            </span>
          </span>
```

To more closely match Figma

Reply:
I'm the worst at styling, thanks.